### PR TITLE
Calculate Average by Arjen van Eerde 9.6 sec

### PR DESCRIPTION
**Pull request by arjenvaneerde**
#### Check List:
- [x] Tests pass (`./test.sh <username>`) All tests fail, though I suspect differences because of \r\n and \n diffs. Visually, all is the same.
- [x] All formatting changes by the build are committed
- [x] Output matches that of calculate_average_baseline.sh
* Execution time: 9.6 sec
* Execution time of reference implementation: 142.0 sec

System specs:
Windows 11 laptop, AMD Ryzen 9 5900HX, 3301 MHz, 8 cores, 16 logical cores, 32GBRAM

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Sorry, all tests need to pass before we can further evaluate this.

**Comment by gunnarmorling** [comment]
Can you also make the launch script executable, please.

**Comment by arjenvaneerde** [comment]
Test issues were due to LF / CRLF conversion issues. After solving that I was able to run tests. After solving some issues, all tests run fine now.

File: src/main/java/dev/morling/onebrc/CalculateAverage.java [hunk 1]
```diff
@@ -52,6 +52,7 @@ private static class MeasurementAggregator {
     }
 
     public static void main(String[] args) throws IOException {
+        long startTime = System.currentTimeMillis();
```
Comment (gunnarmorling) [inline src/main/java/dev/morling/onebrc/CalculateAverage.java:55]: I'll revert these changes, as they shouldn't be here. LGTM otherwise.

**Comment by gunnarmorling** [comment]
00:16.823, nice. Thx for participating!

**Pull request closed without merging**